packages/eslint-plugin/src/configs/all.ts

```diff
@@ -146,7 +146,34 @@ export const all: Linter.BaseConfig = {
             "ts-check": false,
           },
         ],
+        // ported from tslint builtin rules
+        "@typescript-eslint/adjacent-overload-signatures": "error",
+        "@typescript-eslint/ban-types": [
+          "error",
+          {
+            types: { "{}": false },
+            extendDefaults: true,
+          },
+        ],
         "@typescript-eslint/prefer-namespace-keyword": "error",
+        "@typescript-eslint/triple-slash-reference": ["error", { types: "prefer-import", path: "always" }],
+        "@typescript-eslint/no-empty-interface": "error",
+        "no-duplicate-imports": "error",
+        "@typescript-eslint/array-type": ["error", { default: "array-simple" }],
+        "@typescript-eslint/naming-convention": [
+          "error",
+          {
+            selector: "interface",
+            format: [],
+            custom: {
+              regex: "^I[A-Z]",
```

Comment:
Which rule provided this before? Is it definitely this regex? This seems kinda strict but maybe we already pass with it.

Reply:
It's directly based on the recommended port from `interface-name-prefix` documented on the ts-eslint site:
https://typescript-eslint.io/rules/naming-convention#enforce-that-interface-names-do-not-begin-with-an-i